fem/estimators.cpp

```diff
@@ -358,91 +339,67 @@ void KellyErrorEstimator::ComputeEstimates()
 
    pflux->ExchangeFaceNbrData();
 
-   for (int sf = 0; sf < pmesh->GetNSharedFaces(); sf++)
+   for (int sfi = 0; sfi < pmesh->GetNSharedFaces(); sfi++)
    {
-      auto FT = pmesh->GetSharedFaceTransformations(sf, true);
+      auto FT = pmesh->GetSharedFaceTransformations(sfi, true);
+
       if (attributes.Size() &&
           (attributes.FindSorted(FT->Elem1->Attribute) == -1
            || attributes.FindSorted(FT->Elem2->Attribute) == -1))
       {
          continue;
       }
 
-      auto &int_rule = IntRules.Get(FT->FaceGeom, 2 * xfes->GetFaceOrder(0));
+      // TODO Refactor this computation into a user-facing function.
+      // auto &int_rule = IntRules.Get(FT->FaceGeom, 2 * xfes->GetFaceOrder(0)); // NOTE: This fails for DG
```

Comment:
As above, I suggest fixing GetFaceOrder for DG spaces to resolve to the minimum or maximum of the attached face orders.

Reply:
I will take a look, but I might need a few days.

Reply:
I cannot reproduce the crash after merging master. In addition I added a test to catch regression on this. However, I do not have a query for the variable order case here, because I cannot find a good angle on fixing GetFaceOrder on the variable order case. This will also require additional test coverage. Probably this should be fixed in a separate PR, because this PR was merely proposed to fix the faulty scaling of the error.